build/ts/package.json

```diff
@@ -26,5 +26,5 @@
     "@protobuf-ts/plugin": "^2.9.0",
     "glob": "^10.3.3"
   },
-  "main": "/ts/types"
```

Comment:
also point main to an actual model. trying to figure out how to get this module into denoland.